src/sentry/static/sentry/app/views/organizationIntegrationConfig.jsx

```diff
@@ -0,0 +1,305 @@
+import {Flex, Box} from 'grid-emotion';
+import React from 'react';
+import styled from 'react-emotion';
+import _ from 'lodash';
+
+import {sortArray} from '../utils';
+import {t} from '../locale';
+import AlertLink from '../components/alertLink';
+import AsyncView from './asyncView';
+import Button from '../components/buttons/button';
+import Confirm from '../components/confirm';
+import EmptyMessage from './settings/components/emptyMessage';
+import IndicatorStore from '../stores/indicatorStore';
+import LoadingError from '../components/loadingError';
+import Panel from './settings/components/panel';
+import PanelBody from './settings/components/panelBody';
+import PanelHeader from './settings/components/panelHeader';
+import PanelItem from './settings/components/panelItem';
+import PluginIcon from '../plugins/components/pluginIcon';
+import SettingsPageHeader from './settings/components/settingsPageHeader';
+import marked from '../utils/marked';
+
+const IntegrationIcon = styled.img`
+  height: 32px;
```

Comment:
not your fault, but some of these px values make me think we really need some abstraction for managing them